lms/services/user.py

```diff
@@ -227,10 +228,8 @@ def get_users(  # noqa: PLR0913, PLR0917
 
         return (
             select(User)
-            .where(
-                User.id.in_(query)
-                # We can sort these again without affecting deduplication
-            )
+            .where(User.id.in_(query))
```

Comment:
Just a change of format here, move the comment about sorting next to the actual sorting.